ksqldb-parser/src/main/antlr4/io/confluent/ksql/parser/SqlBase.g4

```diff
@@ -127,7 +127,11 @@ tableElements
     ;
 
 tableElement
-    : identifier type ((PRIMARY)? KEY)?
+    : identifier type (((PRIMARY)? KEY) | headerType)?
+    ;
+
+headerType
+    : HEADERS
     ;
```

Comment:
This type seems lonely. Could the `HEADERS` keyword be part of the `tableElement` instead of `headerType` (like the PRIMARY KEY)? Or are there other reasons to keep this?